llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -952,6 +957,44 @@ class StoreInst final : public Instruction {
 #endif
 };
 
+class UnreachableInst final : public Instruction {
+  /// Use UnreachableInst::create() instead of calling the constructor.
+  UnreachableInst(llvm::Instruction *I, Context &Ctx)
```

Comment:
The first argument should be an `llvm::UnreachableInst *`.

Reply:
fixed it.